src/components/icon/icon.component.tsx

```diff
@@ -156,6 +159,7 @@ const Icon = React.forwardRef<HTMLSpanElement, IconProps>(
       disabled: disabledFromContext || disabled,
       fontSize,
       hasTooltip,
+      hideAriaLabel,
```

Comment:
**question**: I'm probably being stupid again here - but why does `hideAriaLabel` need to go down to the underlying styled component (a `span` in this case)? It doesn't appear to use it, and no styles will depend on it as far as I can see - nor is it a valid HTML attribute.